content/rs/references/crdb-cli-reference.md

```diff
@@ -6,8 +6,8 @@ alwaysopen: false
 categories: ["RS"]
 ---
 An [Active-Active database]({{< relref "/rs/administering/active-active.md" >}}) (also known as CRDB or Conflict-free, Replicated DataBase)
-replicates your dataset across multiple, geographically distributed regions and allows read-write access in all locations.
-Active-Active databases are ideal for distributed applications that require the fastest response times, and also for disaster recovery.
+replicates your dataset across Redis Enterprise Software (RS) clusters that that are in geographically distributed regions.
+Active-Active databases allow read-write access in all locations so they are ideal for distributed applications that require the fastest response times, and also for disaster recovery.
```

Comment:
Active-Active databases allow read-write access in all locations, making them ideal....